truffle-config.js

```diff
@@ -1,8 +1,8 @@
 module.exports = {
-  // Uncommenting the defaults below 
+  // Uncommenting the defaults below
   // provides for an easier quick-start with Ganache.
   // You can also follow this format for other networks;
-  // see <http://truffleframework.com/docs/advanced/configuration>
+  // see <https://trufflesuite.com/docs/truffle/reference/configuration>
```

Comment:
I didn't notice this before, but those angle brackets look weird for commented code. How about `See details at:  http...`

Reply:
Okay, I think this will be better. I will make the changes.